infrastructure/github.tf

```diff
@@ -0,0 +1,55 @@
+resource "google_service_account" "github_actions" {
+  account_id   = "github-actions"
+  display_name = "github-actions"
+}
+
+resource "google_artifact_registry_repository" "main" {
+  provider = google-beta
```

Comment:
Wondering why the provider is set to 'google-beta' instead of the 'google' provider?

Reply:
👍